lib/analyze_hoc_activity_helper.rb

```diff
@@ -63,7 +63,13 @@ def rank_tutorials(end_date)
     memo[key] = {}
     start_date = end_date - num_days.days
     PEGASUS_REPORTING_DB_READONLY.fetch(
-      "SELECT tutorial, #{WEIGHTED_COUNT} #{generate_from_where(start_date, end_date: end_date, tutorial_pixel: false)} GROUP BY tutorial ORDER BY count DESC"
+      "SELECT tutorial, COUNT(*) AS count
+        FROM hoc_learn_activity
+        WHERE (
+          created_at >= '#{start_date.strftime('%Y-%m-%d')}' AND
+          created_at < '#{end_date.strftime('%Y-%m-%d')}'
+        )
+        GROUP BY tutorial ORDER BY count DESC"
```

Comment:
This seems wrong, with the reason depending on the general plan.

If we are not backfilling the `hoc_learn_activity` table, this doesn't count historic tutorials.

If we are backfilling the `hoc_learn_activity` table, this seems to unnecessarily copy 210M rows and defeat the purpose of a "temporary" table.